services/src/datasets/external/netcdfcf/mod.rs

```diff
@@ -125,6 +124,34 @@ impl<D: GeoEngineDb> DataProviderDefinition<D> for NetCdfCfDataProviderDefinitio
     }
 }
 
+impl NetCdfCfDataProviderDefinition {
+    async fn _initialize(self: Box<Self>) -> crate::error::Result<NetCdfCfDataProvider> {
+        let mut pg_pool_builder = Pool::builder();
+        if self.metadata_db_config.schema == "pg_temp" {
```

Comment:
can this be done during deserialization of DatabaseConnectionConfig?

Reply:
The deserialization does not set the number of workers, right?